Dockerfile

```
@@ -0,0 +1,3 @@
+FROM golang:latest
+# We need OpenSSL headers to build the simulator
+RUN apt-get update && apt-get install -y libssl-dev
```

Would suggest to remove apt cache to minimize the image. No need to have information about all the available packages in the final image. 😅 

```suggestion
RUN apt-get update && apt-get install -y libssl-dev && rm -rf /var/lib/apt/lists/*
```

---

Thanks! Fixed here and in https://github.com/google/go-tpm-tools/pull/42